experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -58,15 +60,35 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
   private readonly _spanNotEnded: WeakSet<Span> = new WeakSet<Span>();
   private readonly _version = process.versions.node;
   private _headerCapture;
+  private _httpServerDurationHistogram!: Histogram;
+  private _httpClientDurationHistogram!: Histogram;
 
   constructor(config?: HttpInstrumentationConfig) {
     super(
       '@opentelemetry/instrumentation-http',
       VERSION,
       config
     );
-
     this._headerCapture = this._createHeaderCapture();
+    this._updateMetricInstruments();
+  }
+
+  override setMeterProvider(meterProvider: MeterProvider) {
+    super.setMeterProvider(meterProvider);
+    this._updateMetricInstruments();
+  }
+
+  private _updateMetricInstruments() {
+    this._httpServerDurationHistogram = this.meter.createHistogram('http.server.duration', {
```

Comment:
Metric Semantic Conventions are still not available in @opentelemetry/semantic-conventions
, blocked by https://github.com/open-telemetry/opentelemetry-specification/issues/2666

Reply:
That's ok for the sake of this PR. Would be nice to improve later.